packages/components/avatar/src/Avatar/Avatar.tsx

```diff
@@ -23,9 +21,10 @@ export interface AvatarProps extends CommonProps {
    */
   isLoading?: boolean;
   /**
+   * Use the available sizes or a numerical custom one, e.g. '52' or '52px'
    * @default 'medium'
    */
-  size?: Size;
+  size?: Size | string;
```

Comment:
I would rather adhere to the sizes we have in our product than introduce this to break the rules 🤷

Reply:
Without having a deep understanding of it, I’m not seeing the value in allowing overrides for images and skeletons. Could you provide an example of when these overrides would be necessary?

Reply:
The account setting page uses the Avatar component, with a custom size via CSS override, which results in this 
https://github.com/user-attachments/assets/2cfe3e24-d050-4745-a06d-739236e8bb25

With the proposed changes, the size can be specified via the Avatar props (e.g. `52px`), does not require CSS override, and is propagated to the Image + skeleton loader components.

Reply:
> I would rather adhere to the sizes we have in our product than introduce this to break the rules 🤷

I generally agree with that. Most other components don't allow that kind of override, but we limit users to specific values.

In this case, CSS overrides are not straight forward. The size impacts multiple CSS properties and relies on knowing the internal structure.

With the proposed type, we loose all autocompletion for the property. I would therefore go for `Size | `${number}px` or similar. That way, the IDE still suggests the official sizes.

Alternatively, we could allow passing a custom size but hide that from the public API.

Reply:
Good point for the typing, template literal is a better approach.